packages/dds/tree/src/core/rebase/utils.ts

```diff
@@ -74,6 +74,15 @@ export interface BranchRebaseResult<TChange> {
 	 * Details about how the commits on the source branch changed
 	 */
 	readonly commits: RebasedCommits<TChange>;
+	/**
+	 * The length of the source branch after the rebase.
+	 */
+	readonly sourceBranchLength?: number;
```

Comment:
Let's be sure to document what it means for these properties to be undefined.

Reply:
@Josmithr 

`undefined` since these properties are optional right (i.e., properties are only _provided_ when asked for telemetry).

Reply:
@noencke 

Great point. I believe the properties should be provided altogether. The values aren't dependent on each other. However, I think it makes sense to return everything all at once since these are associated with the telemetry.